### DISCUSSION TITLE
Multi Language Project - Autocomplete not appearing

### DISCUSSION THREAD

**Discussion by JonathanHo16** [zed-industries/zed#8842] [Feature Requests]
Hi, 
I have a project, that has both a frontend and backend component in two different languages. When I open the whole project containing both the front and backend folders the autocomplete doesn't appear. Is this possible? or do I not have Zed set up correctly? How can I set up zed  to achieve this?

**Comment by Moshyfawn** [comment]
I believe there's an existing issue #5253 that you want to +👍.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
